tests/replication/slave/storage/test_events.py

```diff
@@ -230,10 +230,12 @@ def persist(
             state_handler = self.hs.get_state_handler()
             context = yield state_handler.compute_event_context(event)
 
-        for user_id, actions in push_actions:
-            yield self.master_store.add_push_actions_to_staging(
-                event.event_id, user_id, actions,
-            )
+        yield self.master_store.add_push_actions_to_staging(
+            event.event_id, {
```

Comment:
isn't this the same as just passing `push_actions`? or are you deliberately copying?

Reply:
The push actions are lists of tuples, rather than dicts here, I think? We could go and change the function to take the push_actions as a dict.

Reply:
Nobody knows, because it's not documented. But yes, I see what you mean.